src/main/java/graphql/schema/DataFetcher.java

```diff
@@ -3,5 +3,5 @@
 
 public interface DataFetcher {
```

Comment:
Anyway, I think I'd prefer to leave this for a different PR. This one is already pretty big IMO.